extensions/assets-manager/AssetsManagerEx.cpp

```diff
@@ -314,7 +314,8 @@ void AssetsManagerEx::setStoragePath(const std::string& storagePath)
     _fileUtils->createDirectory(_storagePath);
     
     _tempStoragePath = _storagePath;
-    _tempStoragePath.insert(_storagePath.size() - 1, TEMP_PACKAGE_SUFFIX);
+    _tempStoragePath.insert(_storagePath.size(), TEMP_PACKAGE_SUFFIX);
```

Comment:
Then use `string::append` is better? And change `TEMP_PACKAGE_SUFFIX ` to `temp` is better too?

Reply:
I have changed!